mail_prevent_bounce_loop/models/res_partner.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    no_bounce_email = fields.Boolean(
+        help="When checked, bounced emails will not be forwarded to this address"
+    )
+    is_automatic_reply_address = fields.Boolean(
```

Comment:
I find this name a little misleading, better use one that expresses what this *does*, as in `no_chatter_forward` or similar